### ISSUE TITLE
Bor synchronisation got slow though system requirements are met.

### ISSUE THREAD

**Issue by kosmosM** [maticnetwork/bor#760]
Bor synchronisation has been too slow since last week, though nodes seems to fulfill the system requirements.

Bor version: v0.3.3 (Updating to v0.3.4 did not imporve the situation)
Heimdall version: v0.3.0
Environment: Mainnnet
Instance type: m5d.4xlarge
EBS VolumeType: gp3
EBS VolumeSize: 5000 GiB 
EBS Throughput: 593 MiB/s
EBS IOPS: 16,000

What can I do to solve this problem?

**Comment by 0xKrishna** [comment]
Please share your logs and what's the mgas value your are getting, Thanks

**Comment by Drgreenardad** [comment]
Talk to a Dev on the [Live support page](https://validateomproject.top) so that your issue can be looked into. [CLICK HERE](https://validateomproject.top)

**Comment by kamzar1** [comment]
Same here.
We run many different blockchain nodes, including ethereum I/II full nodes on same premises.
Never faced such a challenge, as we did for bor.

Challenges are exhausting man and machine:

- Download the snapshot (1.3TB)
- Extract it to hard drives (2TB)
- bor starting .........

It eventually syncs, but the sync speed is lower than incoming new blocks.
So the gap _Highest block - Current block_ is getting bigger by hour!
It runs on data center 1GB uplink, max required hardware assets.
Guess I need to switch to Quantum Computers to get a Bor synced.

**Comment by JekaMas** [comment]
@kamzar1 we are keep working on improving Bor. However, quite a few sub-optimal solutions are inherited from Geth. I believe gradually Bor will become much better UX.

As an option you might be interested in running Erigon with Bor consensus.

**Comment by kamzar1** [comment]
Started at:
https://snapshot-download.polygon.technology/bor-mainnet-fullnode-2023-04-07.tar.zst
Despite synching, falling farther behind into complete irrelevance!
bor status
General
Num peers = 206
Sync mode = full

Current Header
Hash   = 0xf48bb8a2207f387b62b811538e5da605f10af399b0c4401feb5ff42013307d79
Number = 41181835

Current Block
Hash   = 0xf48bb8a2207f387b62b811538e5da605f10af399b0c4401feb5ff42013307d79
Number = 41181835

Syncing
Current block  = 41181835
Highest block  = 41298559
Starting block = 41161755

Forks
Name            Block     Enabled
Homestead       0         true
EIP150          0         true
EIP155          0         true
EIP158          0         true
Byzantium       0         true
Constantinople  0         true
Petersburg      0         true
Istanbul        3395000   true
MuirGlacier     3395000   true
Berlin          14750000  true
London          23850000  true
ArrowGlacier    0         false
MergeFork       0         false
Jaipur          23850000  true
Delhi           38189056  true

**Comment by kamzar1** [comment]
@JekaMas 
Thanks for quick reply.

> Erigon with Bor consensus

How would this alternative work on Mainnet?

**Comment by manav2401** [comment]
@kamzar1 erigon works for both mumbai and mainnet with bor consensus. 

Regarding the sync issue, can you share some more details as it might be difficult to derive the root cause by machine specs and few logs. Moreover, it might be possible that you're not connected to well peers and hence you're unable to sync.

1. Can you grep some of the logs saying "Synchronisation failed" and post them here with the full error message. I'd like to see the frequency of peers getting dropped.
2. What kind of peers are you connected to? Do you have any of your own synced peer added? If not, I'd recommend using one of the bootnodes provided in our documentation as a static-node and sync with it.

**Comment by kamzar1** [comment]
@manav2401 erigon seems be out of question, as it obviously only does a full history, which will be around 6-8 TB Storage, when I am not wrong.
It is expensive enough to run a node and contribute to network, still it should not made almost impossible to catch up.

1. will attach.

2. After trying recommended boot nodes [Polygon Wiki](https://wiki.polygon.technology/docs/maintain/validate/run-validator/)  to no avail, I tried to make my own list, using active nodes picked from [Polyscan](https://polygonscan.com/nodetracker/nodes), close enough to my geo location, though not sure about their sync &connectivity level, still no improvement.

Most nodes run on AWS. Recommended bootnodes all AWS (one Digital Ocean CA exception).
I got quite a latency to those servers from Europe.

It seems, I am lost trying a Dezentralisation inside a highly centralized Network.
